browser/ui/tabs/features.cc

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/tabs/features.h"
+
+#include "base/feature_list.h"
+
+namespace tabs {
+namespace features {
+
+const base::Feature kBraveVerticalTabs{"BraveVerticalTabs",
```

Comment:
re https://github.com/brave/brave-core/pull/13816/files#r934749501 - it seems like we should enable/disable by channel here?

Reply:
the closest thing I could find to this was `BorealisFeatures::MightBeAllowed` which would actually mean keeping `IsVerticalTabsEnabled`, but adding the channel check there

Reply:
I'm not really sure what the best option is here. @iefremov @goodov any opinions? The expire thing seems weird to me.

Reply:
I prefer Griffin for this type of activity.